scripts/download_onDownloadProgress.js

```diff
@@ -0,0 +1,68 @@
+/**
```

Comment:
I would try to integrate the progress options into the existing download and upload scripts. I don't think we need standalone scripts for this. If that doesn't work, out you can leave these scripts; but rename them to e.g. `download_progress.js` please.

Reply:
Hello **m-cat,**

I'm still thinking about what makes more sense.
I could integrate it into the download script, **but I have problems implementing it with the upload script.**

**I myself think it's better to allow the scripts in your own files, because you can only test the "onUploadProgress" function if the file is larger than 41mb.**

And I think I have to give a hint in the "upload_progress.js" if the size of the file is smaller than 41MB, so that no progress can be seen.

More information:
The "onUploadProgress Tracker" is the original from the sdk-doc.
Since I didn't like the original one, I created two others that I think look nicer in the console.
I installed it in my new CLI.
If you don't like the original onUploadProgress Tracker, we can include one of mine.